00-Getting_Started/get_inspektor_gadget.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Check that kubectl is installed
+if ! kubectl version >/dev/null 2>&1; then
```

Comment:
```suggestion
if ! kubectl version --client=true >/dev/null 2>&1; then
```

In this way, kubectl will not try to connect to the Kubernetes API Server: Minikube might not be installed at this stage.